src/components/analyser/details/DetailsHeader.tsx

```diff
@@ -91,6 +96,8 @@ const DetailsHeader: React.FC<DetailsProps> = ({ details, back }) => {
               size="35px"
               paintJob="#EEF1FB"
             />
+            <>&nbsp;</>
+            <InfoButton infotext="Very Cheap must buy!!!" />
```

Comment:
Kein InfoButton nötig

Reply:
In den Info Button sollte eig sowas wie Last Price: 31.03.21 20:00 odersowas hatte ich gedacht kann ma aber auch weg lassen...